### PR TITLE
Update Gymnasium to v1.0.0

### PR DESCRIPTION
**Pull request by LucasAlegre**
From Gymnasium's release notes (alpha 1 & 2) https://github.com/Farama-Foundation/Gymnasium/releases 👍 

- [x] Bump lunar lander to v3
- [x] Bump highway env when they do the bump to gymnasium v1
- [x] New vectorized envs (+ check MORL-Baselines' PGMORL) https://github.com/LucasAlegre/morl-baselines/pull/109
- [x] New wrappers

### CODE REVIEW

File: mo_gymnasium/wrappers/wrappers.py [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+    def step(self, action: ActType):
+        """Steps through the environment, normalizing the rewards returned.
+
+        Args:
+            action: action to perform
+        Returns: obs, normalized_rewards, terminated, truncated, infos
+        """
+        obs, rews, terminated, truncated, infos = self.env.step(action)
+        # Extracts the objective value to normalize
+        print("Normalizing reward at index", self.idx)
```
Comment (pseudo-rnd-thoughts) [inline mo_gymnasium/wrappers/wrappers.py:101]: Remove

**Comment by LucasAlegre** [comment]
Currently, the only test not passing is the one related to highway-env. This should be fixed when https://github.com/Farama-Foundation/HighwayEnv/pull/609 is merged.

**Comment by LucasAlegre** [comment]
Looks good to me now! :)

**Review by ffelten — APPROVED** [review]
LGTM

**Comment by LucasAlegre** [comment]
@ffelten can you fix these conflicts?

**Comment by ffelten** [comment]
> @ffelten can you fix these conflicts?

I was doing it x)

**Pull request merged**